main.js

```diff
@@ -0,0 +1,17 @@
+document.querySelector("li").remove();
+
+document.querySelector("span").remove();
+
+document.querySelector(".posts article").remove();
```

Comment:
Close, but we want the last post on the page. This will remove the first non-featured post on the page.